### ISSUE TITLE
Modify existing GCC 4.8.4 CI build to match selected auto PR build

### ISSUE THREAD

**Issue by bartlettroscoe** [trilinos/Trilinos#2462]
CC: @trilinos/framework, @mhoemmen, @rppawlo, @ibaned, @crtrott 

May tests timeout when trying to upgrade from OpenMPI 1.6.5 to OpenMPI 1.8.7 (30 timeouts) or OpenMPI 1.10.1 (34 timeouts).  Next: Debug why these tests are timing out?

## Next Action Status

Post-push CI build and checkin-test-sems.sh script is now updated to use updated GCC 4.8.4 + OpenMPI 1.10.1 + OpenMP build.  Next close this issue and open a new issue to have this used as the auto PR GCC 4.8.4 build?

## Description

This Issue is to scope out and track efforts to upgrade the existing SEMS-based Trilinos CI build (see #482 and #1304) to match the selected GCC 4.8.4 auto PR build as described in https://github.com/trilinos/Trilinos/issues/2317#issuecomment-376551457.  The existing GCC 4.8.4 CI build shown [here](https://github.com/trilinos/Trilinos/wiki/Policies--%7C-Testing#post_push_ci_testing) has been running for 1.5+ years and has been maintained over that time.  That build has many but not all of the settings of the selected GCC 4.8.4 auto PR build listed [here](https://docs.google.com/document/d/1bCqOJLoXSnq24ct-IjwGkvse6xhsp9HrHsxqZxBwDWQ/edit#bookmark=id.3idxwkmmlqcg).  The primary changes that need to be made are:

* Switch from OpenMPI 1.6.5 to 1.8.7.
* Enable `Xpetra_ENABLE_Experimental=ON` and `MueLu_ENABLE_Experimental=ON`  (note objection in https://github.com/trilinos/Trilinos/issues/2317#issuecomment-376575762).
* Enable OpenMP (`OMP_NUM_THREADS=2`).

The most difficult change will likely be to enable OpenMP because of the problem of the threads all binding to the same cores as described in #2422.  Therefore, the initial auto PR build may not have OpenMP enabled due to these challenges.

## Tasks:

1. Set `Xpetra_ENABLE_Experimental=ON` and `MueLu_ENABLE_Experimental=ON` in CI build ... Merged in #2467 and was later removed in 7481c760699d8b0c30034782cb2ef0c742ce6657  **[DONE]**
1. Switch current CI build from OpenMPI 1.6.5 to 1.10.1 (see build `GCC-4.8.4-OpenMPI-1.10.1-MpiReleaseDebugSharedPtOpenMP` in #2688) **[DONE]**
1. Enable `Trilinos_ENABLE_OpenMP=ON` and `OMP_NUM_THREADS=2` (see build `GCC-4.8.4-OpenMPI-1.10.1-MpiReleaseDebugSharedPtOpenMP` in #2688) **[DONE]**
1. Set up nightly build and clean up tests (see #2691 and #2712) ... **IN PROGRESS** ...
1. Switch auto PR tester to use updated GCC 4.8.4 configuration ...

## Related Issues:

* Part of: #2317 
* Blocked by: #2691, #2712

**Comment by mhoemmen** [comment]
@bartlettroscoe Can we enable OpenMP but force `OMP_NUM_THREADS=1`?  Some of those Xpetra and MueLu "experimental" build options may not have any effect unless OpenMP is enabled.

**Comment by bartlettroscoe** [comment]
> Can we enable OpenMP but force `OMP_NUM_THREADS=1`? Some of those Xpetra and MueLu "experimental" build options may not have any effect unless OpenMP is enabled.

I guess I can try that.  But I wonder even with  `OMP_NUM_THREADS=1` if all of the threads will be bound to the same core or not.

Also, note that there are ATDM builds of Trilinos that enable experimental MueLu code that build and run tests with a serial Kokkos node as shown at:

* https://testing.sandia.gov/cdash/index.php?project=Trilinos&filtercount=1&showfilters=1&field1=buildname&compare1=63&value1=-atdm-

**Comment by mhoemmen** [comment]
@csiefer2 would know for sure whether disabling OpenMP is adequate.  My guess is no, because some of the sparse matrix-matrix multiply code takes different paths if OpenMP is enabled.

**Comment by csiefer2** [comment]
OpenMPNode and SerialNode trigger different code paths in chunks of Tpetra. AFAIK MueLu does not do node type specialization (except for Epetra).

What you choose to test for PR doesn't really matter, but they both need to stay working (more or less).

**Comment by bartlettroscoe** [comment]
> OpenMPNode and SerialNode trigger different code paths in chunks of Tpetra. AFAIK MueLu does not do node type specialization (except for Epetra).
> 
> What you choose to test for PR doesn't really matter, but they both need to stay working (more or less).

The GCC 4.8.4 PR build will test OpenMP path and Intel 17.x build will test the Serial node path.  And the ATDM builds of Trilinos are already testing both paths and have been for many weeks now as you can see at:

* https://snl-wiki.sandia.gov/display/CoodinatedDevOpsATDM/ATDM+Builds+of+Trilinos

**Comment by mhoemmen** [comment]
@bartlettroscoe Cool, then I'm OK with this :)

**Comment by bartlettroscoe** [comment]
I submitted PR #2467 to enable Xpetra and MueLu experimental code in the standard CI build.  If someone can quickly review that, then I can merge.

**Comment by bartlettroscoe** [comment]
I tested the full CI build going from OpenMPI 1.6.5 to 1.8.7 in the branch `2462-openmpi-1.6.5-to-1.8.7` in my fork of Trilinos `git@github.com:bartlettroscoe/Trilinos.git` and it caused 30 tests to time out (see details below).  I can't tell if these are hangs or just that MPI communication is taking longer.   Someone would need to research that.  In any case, we are a no-go for upgrading from OpenMPI 1.6.5 to 1.8.7.

I will try updating from OpenMPI 1.6.5 to 1.10.1 (which is the only other OpenMPI implementation that SEMS provides) and see how that goes.

<details>
<summary>
  <b>DETAILED NOTES</b> (click to expand)
</summary>

**(3/27/2018)**

I created the branch `2462-openmpi-1.6.5-to-1.8.7` in my fork of Trilinos.   I added the commit d36479f to change from OpenMPI 1.6.5 to 1.8.7. 

I tested this with:

```
$ ./checkin-test-sems.sh --enable-all-packages=on --local-do-all
```

and it returned:

```
FAILED: Trilinos/MPI_RELEASE_DEBUG_SHARED_PT: passed=2557,notpassed=30

Tue Mar 27 19:21:00 MDT 2018

Enabled Packages: 
Disabled Packages: PyTrilinos,Claps,TriKota
Enabled all Packages
Hostname: crf450.srn.sandia.gov
Source Dir: /home/rabartl/Trilinos.base/Trilinos/cmake/tribits/ci_support/../../..
Build Dir: /home/rabartl/Trilinos.base/BUILDS/CHECKIN/MPI_RELEASE_DEBUG_SHARED_PT

CMake Cache Varibles: -DTrilinos_TRIBITS_DIR:PATH=/home/rabartl/Trilinos.base/Trilinos/cmake/tribits -DTrilinos_ENABLE_TESTS:BOOL=ON -DTrilinos_TEST_CATEGORIES:STRING=BASIC -DTrilinos_ALLOW_NO_PACKAGES:BOOL=OFF -DDART_TESTING_TIMEOUT:STRING=300.0 -DBUILD_SHARED_LIBS=ON -DTrilinos_DISABLE_ENABLED_FORWARD_DEP_PACKAGES=ON -DTrilinos_ENABLE_SECONDARY_TESTED_CODE:BOOL=OFF -DTrilinos_CONFIGURE_OPTIONS_FILE:STRING=cmake/std/MpiReleaseDebugSharedPtSettings.cmake,cmake/std/BasicCiTestingSettings.cmake,cmake/std/sems/SEMSDevEnv.cmake -DTrilinos_ENABLE_ALL_OPTIONAL_PACKAGES:BOOL=ON -DTrilinos_ENABLE_ALL_PACKAGES:BOOL=ON -DTrilinos_ENABLE_ALL_FORWARD_DEP_PACKAGES:BOOL=ON -DTrilinos_ENABLE_PyTrilinos:BOOL=OFF -DTrilinos_ENABLE_Claps:BOOL=OFF -DTrilinos_ENABLE_TriKota:BOOL=OFF
Make Options: -j16
CTest Options: -j16 

Pull: Not Performed
Configure: Passed (3.29 min)
Build: Passed (92.01 min)
Test: FAILED (26.13 min)

99% tests passed, 30 tests failed out of 2587

Label Time Summary:
Amesos               =   6.08 sec (14 tests)
Amesos2              =   3.33 sec (9 tests)
Anasazi              =  34.87 sec (71 tests)
AztecOO              =   5.50 sec (17 tests)
Belos                =  35.43 sec (72 tests)
Domi                 = 138.39 sec (125 tests)
Epetra               =  43.01 sec (61 tests)
EpetraExt            =   4.18 sec (11 tests)
FEI                  =  51.74 sec (43 tests)
Galeri               =   7.70 sec (9 tests)
GlobiPack            =   3.08 sec (6 tests)
Ifpack               =  26.56 sec (53 tests)
Ifpack2              =  54.12 sec (35 tests)
Intrepid             = 1467.90 sec (152 tests)
Intrepid2            = 336.96 sec (144 tests)
Isorropia            =   1.74 sec (6 tests)
Kokkos               = 379.99 sec (23 tests)
KokkosKernels        = 701.31 sec (4 tests)
ML                   =  25.33 sec (34 tests)
MiniTensor           =   0.72 sec (2 tests)
MueLu                = 1217.24 sec (84 tests)
NOX                  = 413.98 sec (106 tests)
OptiPack             =   2.69 sec (5 tests)
Panzer               = 808.72 sec (154 tests)
Phalanx              =  21.44 sec (27 tests)
Pike                 =   4.37 sec (7 tests)
Piro                 =  25.49 sec (12 tests)
ROL                  = 3183.59 sec (153 tests)
RTOp                 =  11.01 sec (24 tests)
Rythmos              = 1083.47 sec (83 tests)
SEACAS               =  50.54 sec (14 tests)
STK                  = 109.42 sec (12 tests)
Sacado               = 122.29 sec (292 tests)
Shards               =   1.77 sec (4 tests)
ShyLU_Node           =   2.29 sec (3 tests)
Stokhos              = 436.12 sec (75 tests)
Stratimikos          = 167.50 sec (40 tests)
Teko                 = 362.38 sec (19 tests)
Tempus               = 7650.88 sec (36 tests)
Teuchos              =  50.89 sec (137 tests)
ThreadPool           =   5.48 sec (10 tests)
Thyra                =  35.63 sec (81 tests)
Tpetra               =  71.71 sec (162 tests)
TrilinosCouplings    = 335.97 sec (24 tests)
Triutils             =   0.38 sec (2 tests)
Xpetra               =  27.11 sec (18 tests)
Zoltan               =  26.70 sec (19 tests)
Zoltan2              =  60.17 sec (101 tests)

Total Test time (real) = 1567.87 sec

The following tests FAILED:
	173 - KokkosKernels_graph_serial_MPI_1 (Timeout)
	1984 - MueLu_UnitTestsTpetra_MPI_1 (Timeout)
	1994 - MueLu_ParameterListInterpreterEpetra_MPI_1 (Timeout)
	1998 - MueLu_ParameterListInterpreterTpetra_MPI_1 (Timeout)
	2099 - Rythmos_BackwardEuler_ConvergenceTest_MPI_1 (Timeout)
	2103 - Rythmos_IntegratorBuilder_ConvergenceTest_MPI_1 (Timeout)
	2129 - Tempus_BackwardEuler_MPI_1 (Timeout)
	2131 - Tempus_BackwardEuler_Staggered_FSA_MPI_1 (Timeout)
	2133 - Tempus_BackwardEuler_ASA_MPI_1 (Timeout)
	2134 - Tempus_BDF2_MPI_1 (Timeout)
	2135 - Tempus_BDF2_Combined_FSA_MPI_1 (Timeout)
	2136 - Tempus_BDF2_Staggered_FSA_MPI_1 (Timeout)
	2138 - Tempus_BDF2_ASA_MPI_1 (Timeout)
	2139 - Tempus_ExplicitRK_MPI_1 (Timeout)
	2140 - Tempus_ExplicitRK_Combined_FSA_MPI_1 (Timeout)
	2141 - Tempus_ExplicitRK_Staggered_FSA_MPI_1 (Timeout)
	2143 - Tempus_ExplicitRK_ASA_MPI_1 (Timeout)
	2145 - Tempus_DIRK_MPI_1 (Timeout)
	2146 - Tempus_DIRK_Combined_FSA_MPI_1 (Timeout)
	2147 - Tempus_DIRK_Staggered_FSA_MPI_1 (Timeout)
	2149 - Tempus_DIRK_ASA_MPI_1 (Timeout)
	2150 - Tempus_HHTAlpha_MPI_1 (Timeout)
	2151 - Tempus_Newmark_MPI_1 (Timeout)
	2154 - Tempus_IMEX_RK_Combined_FSA_MPI_1 (Timeout)
	2155 - Tempus_IMEX_RK_Staggered_FSA_MPI_1 (Timeout)
	2157 - Tempus_IMEX_RK_Partitioned_Combined_FSA_MPI_1 (Timeout)
	2158 - Tempus_IMEX_RK_Partitioned_Staggered_FSA_MPI_1 (Timeout)
	2282 - ROL_test_sol_solSROMGenerator_MPI_1 (Timeout)
	2288 - ROL_test_sol_checkAlmostSureConstraint_MPI_1 (Timeout)
	2320 - ROL_example_burgers-control_example_06_MPI_1 (Timeout)

Errors while running CTest

Total time for MPI_RELEASE_DEBUG_SHARED_PT = 121.44 min
```

Darn, taht is not good.  That is a lot of timeouts.  Now, I can't tell if these are timeouts because things are taking longer or if these are hangs.  Someone would need to research that.

</details>

**Comment by mhoemmen** [comment]
@bartlettroscoe I have heard complaints about OpenMPI 1.8.x bugs.  The OpenMPI web page considers it "retired" -- in fact, the oldest "not retired" version is 1.10.

**Comment by mhoemmen** [comment]
@prwolfe Have you seen issues like this with OpenMPI 1.8.x?

**Comment by bartlettroscoe** [comment]
I tested the full CI build going from OpenMPI 1.6.5 to 1.10.1 in the branch `2462-openmpi-1.6.5-to-1.10.1` in my fork of Trilinos `git@github.com:bartlettroscoe/Trilinos.git` and it caused 34 tests to time out (see details below).  I can't tell if these are hangs or just that MPI communication is taking longer to complete (which is hard to believe).

I am wondering if there is not some problem with the way these tests are using MPI and I am wondering if someone should not dig in and try to debug some of these timeouts to see why they are happening?  Perhpas there are some real defects in the code that these updated versions of OpenMPI are bringing out?

<details>
<summary>
  <b>DETAILED NOTES</b> (click to expand)
</summary>

**(3/28/2018)**

I created the branch `2462-openmpi-1.6.5-to-1.10.1` in my fork of Trilinos `git@github.com:bartlettroscoe/Trilinos.git`.   I added the commit c9e9097 to change from OpenMPI 1.6.5 to 1.10.1.

I tested this with:

```
$ ./checkin-test-sems.sh --enable-all-packages=on --local-do-all
```

and it returned:

```
FAILED: Trilinos/MPI_RELEASE_DEBUG_SHARED_PT: passed=2552,notpassed=34

Wed Mar 28 09:41:16 MDT 2018

Enabled Packages: 
Disabled Packages: PyTrilinos,Claps,TriKota
Enabled all Packages
Hostname: crf450.srn.sandia.gov
Source Dir: /home/rabartl/Trilinos.base/Trilinos/cmake/tribits/ci_support/../../..
Build Dir: /home/rabartl/Trilinos.base/BUILDS/CHECKIN/MPI_RELEASE_DEBUG_SHARED_PT

CMake Cache Varibles: -DTrilinos_TRIBITS_DIR:PATH=/home/rabartl/Trilinos.base/Trilinos/cmake/tribits -DTrilinos_ENABLE_TESTS:BOOL=ON -DTrilinos_TEST_CATEGORIES:STRING=BASIC -DTrilinos_ALLOW_NO_PACKAGES:BOOL=OFF -DDART_TESTING_TIMEOUT:STRING=300.0 -DBUILD_SHARED_LIBS=ON -DTrilinos_DISABLE_ENABLED_FORWARD_DEP_PACKAGES=ON -DTrilinos_ENABLE_SECONDARY_TESTED_CODE:BOOL=OFF -DTrilinos_CONFIGURE_OPTIONS_FILE:STRING=cmake/std/MpiReleaseDebugSharedPtSettings.cmake,cmake/std/BasicCiTestingSettings.cmake,cmake/std/sems/SEMSDevEnv.cmake -DTrilinos_ENABLE_ALL_OPTIONAL_PACKAGES:BOOL=ON -DTrilinos_ENABLE_ALL_PACKAGES:BOOL=ON -DTrilinos_ENABLE_ALL_FORWARD_DEP_PACKAGES:BOOL=ON -DTrilinos_ENABLE_PyTrilinos:BOOL=OFF -DTrilinos_ENABLE_Claps:BOOL=OFF -DTrilinos_ENABLE_TriKota:BOOL=OFF
Make Options: -j16
CTest Options: -j16 

Pull: Not Performed
Configure: Passed (3.04 min)
Build: Passed (99.75 min)
Test: FAILED (26.52 min)

99% tests passed, 34 tests failed out of 2586

Label Time Summary:
Amesos               =   6.20 sec (14 tests)
Amesos2              =   3.17 sec (9 tests)
Anasazi              =  37.92 sec (71 tests)
AztecOO              =   5.27 sec (17 tests)
Belos                =  35.64 sec (72 tests)
Domi                 = 122.76 sec (125 tests)
Epetra               =  42.48 sec (61 tests)
EpetraExt            =   5.19 sec (11 tests)
FEI                  =  49.67 sec (43 tests)
Galeri               =   6.93 sec (9 tests)
GlobiPack            =   3.39 sec (6 tests)
Ifpack               =  27.06 sec (53 tests)
Ifpack2              =  60.88 sec (35 tests)
Intrepid             = 1661.11 sec (152 tests)
Intrepid2            = 360.30 sec (144 tests)
Isorropia            =   1.67 sec (6 tests)
Kokkos               = 491.07 sec (23 tests)
KokkosKernels        = 653.29 sec (4 tests)
ML                   =  21.03 sec (34 tests)
MiniTensor           =   1.04 sec (2 tests)
MueLu                = 1227.70 sec (83 tests)
NOX                  = 415.46 sec (106 tests)
OptiPack             =   2.10 sec (5 tests)
Panzer               = 876.56 sec (154 tests)
Phalanx              =  17.27 sec (27 tests)
Pike                 =   3.69 sec (7 tests)
Piro                 =  20.75 sec (12 tests)
ROL                  = 3246.73 sec (153 tests)
RTOp                 =  11.13 sec (24 tests)
Rythmos              = 992.68 sec (83 tests)
SEACAS               =  56.99 sec (14 tests)
STK                  = 127.17 sec (12 tests)
Sacado               = 115.83 sec (292 tests)
Shards               =   1.84 sec (4 tests)
ShyLU_Node           =   1.30 sec (3 tests)
Stokhos              = 285.61 sec (75 tests)
Stratimikos          = 178.14 sec (40 tests)
Teko                 = 433.87 sec (19 tests)
Tempus               = 7758.58 sec (36 tests)
Teuchos              =  51.55 sec (137 tests)
ThreadPool           =   5.48 sec (10 tests)
Thyra                =  35.87 sec (81 tests)
Tpetra               =  66.82 sec (162 tests)
TrilinosCouplings    = 386.35 sec (24 tests)
Triutils             =   0.41 sec (2 tests)
Xpetra               =  25.48 sec (18 tests)
Zoltan               =  26.04 sec (19 tests)
Zoltan2              =  54.41 sec (101 tests)

Total Test time (real) = 1590.85 sec

The following tests FAILED:
	173 - KokkosKernels_graph_serial_MPI_1 (Timeout)
	1506 - Teko_testdriver_tpetra_MPI_1 (Failed)
	1983 - MueLu_UnitTestsTpetra_MPI_1 (Timeout)
	1993 - MueLu_ParameterListInterpreterEpetra_MPI_1 (Timeout)
	1997 - MueLu_ParameterListInterpreterTpetra_MPI_1 (Timeout)
	2098 - Rythmos_BackwardEuler_ConvergenceTest_MPI_1 (Timeout)
	2102 - Rythmos_IntegratorBuilder_ConvergenceTest_MPI_1 (Timeout)
	2127 - Tempus_ForwardEuler_MPI_1 (Timeout)
	2128 - Tempus_BackwardEuler_MPI_1 (Timeout)
	2129 - Tempus_BackwardEuler_Combined_FSA_MPI_1 (Timeout)
	2130 - Tempus_BackwardEuler_Staggered_FSA_MPI_1 (Timeout)
	2132 - Tempus_BackwardEuler_ASA_MPI_1 (Timeout)
	2133 - Tempus_BDF2_MPI_1 (Timeout)
	2134 - Tempus_BDF2_Combined_FSA_MPI_1 (Timeout)
	2135 - Tempus_BDF2_Staggered_FSA_MPI_1 (Timeout)
	2137 - Tempus_BDF2_ASA_MPI_1 (Timeout)
	2138 - Tempus_ExplicitRK_MPI_1 (Timeout)
	2139 - Tempus_ExplicitRK_Combined_FSA_MPI_1 (Timeout)
	2140 - Tempus_ExplicitRK_Staggered_FSA_MPI_1 (Timeout)
	2142 - Tempus_ExplicitRK_ASA_MPI_1 (Timeout)
	2144 - Tempus_DIRK_MPI_1 (Timeout)
	2145 - Tempus_DIRK_Combined_FSA_MPI_1 (Timeout)
	2146 - Tempus_DIRK_Staggered_FSA_MPI_1 (Timeout)
	2148 - Tempus_DIRK_ASA_MPI_1 (Timeout)
	2149 - Tempus_HHTAlpha_MPI_1 (Timeout)
	2150 - Tempus_Newmark_MPI_1 (Timeout)
	2153 - Tempus_IMEX_RK_Combined_FSA_MPI_1 (Timeout)
	2154 - Tempus_IMEX_RK_Staggered_FSA_MPI_1 (Timeout)
	2156 - Tempus_IMEX_RK_Partitioned_Combined_FSA_MPI_1 (Timeout)
	2157 - Tempus_IMEX_RK_Partitioned_Staggered_FSA_MPI_1 (Timeout)
	2281 - ROL_test_sol_solSROMGenerator_MPI_1 (Timeout)
	2287 - ROL_test_sol_checkAlmostSureConstraint_MPI_1 (Timeout)
	2319 - ROL_example_burgers-control_example_06_MPI_1 (Timeout)
	2327 - ROL_example_parabolic-control_example_03_MPI_1 (Timeout)

Errors while running CTest

Total time for MPI_RELEASE_DEBUG_SHARED_PT = 129.32 min
```

Wow, that is even more test timeouts!  

</details>

**Comment by bartlettroscoe** [comment]
> I have heard complaints about OpenMPI 1.8.x bugs. The OpenMPI web page considers it "retired" -- in fact, the oldest "not retired" version is 1.10.

Okay, given that OpenMPI 1.10 is the oldest version of MPI that is supported, we should try to debug what is causing these timeouts.  I will submit an experimental build to CDash and then we can go from there.

**Comment by prwolfe** [comment]
we had lots of issues with 1.8 - that's why we abandoned it.  Basically it was slow and would not properly place processes. In fact we have had some issues with 1.10 but those responded well to placement directives.

**Comment by mhoemmen** [comment]
I remember the "let's try 1.8 .... oh that was bad let's not" episode :(

**Comment by bartlettroscoe** [comment]
I merged #2467 which enables experimental code in Xpetra and MueLu in the GCC 4.8.4 CI build.

**Comment by bartlettroscoe** [comment]
The PR #2609 provides a single *.cmake file to to define this build.  The auto PR tester driver bash script just needs to source:

```
$ source <trilinos-base-dir>/cmake/load_sems_dev_env.sh
```

and then the `ctest -S <script>` driver script just needs the argument:

```
-C <trilinos-base-dir>/cmake/std/MpiReleaseDebugSharedPtPthread.cmake
```

and that is it. 

The most important settings that we don't want to duplicate all over the place that are included in this file are from the files `SEMSDevEnv.cmake` and `BasicCiTestingSettings.cmake`.

**Comment by bartlettroscoe** [comment]
The post-push CI build linked to from:

* https://github.com/trilinos/Trilinos/wiki/Policies--%7C-Testing#post_push_ci_testing

is now set to the updated GCC 4.8.4 + Intel 1.10.1 + OpenMP build and it finished the initial build this morning of all 53 packages passing all 2722 tests.  And it ran all of these tests in a wall-clock time of 
24m 56s (on 8 cores).

@trililinos/framework, I this this build should be ready to substitute for the existing GCC 4.8.4 auto PR build.  Should we open a new GitHub issue for that?

Otherwise, I am putting this in review.

**Comment by bartlettroscoe** [comment]
Given that issue #2788 exists for using this configuration for that auto PR GCC 4.8.4 build, I am closing this issue #2462 since there is nothing left to do.  This updated configuration is being used in the post-push CI build so we will get an email if there are any failures going forward.
